services/__tests__/optimization.test.js

```diff
@@ -0,0 +1,229 @@
+'use strict';
+
+const optimizationService = require('../optimization');
+const tu = require('../../test/test-utils');
+
+let optimization;
+beforeEach(() => {
+  optimization = optimizationService(tu.mockClient());
+});
+
+describe('getOptimization', () => {
+  test('works', () => {
+    optimization.getOptimization({
+      waypoints: [
+        {
+          coordinates: [2.2, 1.1]
+        },
+        {
+          coordinates: [2.2, 1.1]
+        }
+      ]
+    });
+    expect(tu.requestConfig(optimization)).toEqual({
+      path: '/optimized-trips/v1/mapbox/:profile/:coordinates',
+      method: 'GET',
+      params: {
+        coordinates: '2.2,1.1;2.2,1.1',
+        profile: 'driving'
+      },
+      query: {}
+    });
+  });
+
+  test('it omits queries not supplied', () => {
```

Comment:
Not sure what this description means ... what's being "omitted" in this test?

Reply:
Updated the wording here, but I'm thinking we might not even need this test since most of the others will fail if a stray query gets added anyway.